src/components/pinnable.js

```diff
@@ -18,13 +18,17 @@ AFRAME.registerComponent("pinnable", {
   },
 
   _fireEventsAndAnimate(oldData, force) {
-    // We need to guard against _fireEventsAndAnimate being called during entity initialization,
-    // when the networked component isn't initialized yet.
-    if (this.el.components.networked && this.el.components.networked.data && !NAF.utils.isMine(this.el)) return;
+    // during object initialization, we may not have a networked component, and we
+    // also have no desire to fire pinned events or animate anything
+    if (!this.el.components.networked) {
```

Comment:
@brianpeiris you may have an opinion about this. The failure state of the previous code was during object inflation, when the pinnable component may be instantiated before the networked component. In production right now, `pinned` events do fire during inflation of room objects as a result of this, and we just happen to ignore them because we inflate room objects surprisingly early in the initial load.

Reply:
It would be nice if we didn't do this implicit pinning based on events, but rather did an explicit "pin" operation triggered directly from user actions, but you know that already. Other than that, I have no opinion on this. This seems like a fine change to make.